app/builders/subscription_confirmation_email_builder.rb

```diff
@@ -51,4 +45,23 @@ def body
       #{ManageSubscriptionsLinkPresenter.call(subscriber.address)}
     BODY
   end
+
+  def title
+    return subscriber_list.title unless subscriber_list.url
+
+    "[#{subscriber_list.title}](#{title_url})"
+  end
+
+  def title_url
+    query = {
+      utm_source: subscriber_list.slug,
+      utm_medium: "email",
+      utm_campaign: "govuk-notifications-subscription-confirmation",
+    }.to_query
+
+    url = subscriber_list.url
+    tracked_url = url + (url.include?("?") ? "&" : "?") + query
```

Comment:
I don't think you can actually subscribe to a blank results page so there always should be query params so this should be okay - `tracked_url = url + "&" + query`?

Reply:
You're right, but I think I'll leave it in just to be sure, and in the future we might have subscriber lists without parameters.